app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -143,6 +143,31 @@ async function updateExperienceLevel(currentUserId, articleId, rating, group) {
   }
 }
 
+const adminUnpublishArticle = async (id, username, slug) => {
+  try {
+    const response = await request(`/articles/${id}/admin_unpublish`, {
+      method: 'PATCH',
+      body: JSON.stringify({ id, username, slug }),
+      credentials: 'same-origin',
+    });
+
+    const outcome = await response.json();
+
+    /* eslint-disable no-restricted-globals */
+    if (outcome.message == "success") {
+      window.top.location.href = window.location.origin + outcome.path;
```

Comment:
```suggestion
      window.top.location.assign(`${window.location.origin}${outcome.path}`);
```

this for me works without sandboxing, both in Firefox and in Chrome